core/modules/installer/installer.module

```diff
@@ -0,0 +1,516 @@
+<?php
+/**
+ * @file
+ * Handles installation and updates of contributed projects.
+ */
+
+/**
+ * Implements hook_permission().
+ */
+function installer_permission() {
+  return array(
+    'use installer browser' => array(
+      'title' => t('Access installer interface'),
```

Comment:
Let's change this to `Install and Update modules, themes & layouts`

We're allowing a lot more than just access to the interface! Users with this permission can install and update from here too.